coverage/index.html

```diff
@@ -0,0 +1,2097 @@
+<!DOCTYPE html>
```

Comment:
Coverage reports are generally added to the .gitignore
Otherwise every time you run tests, all of the coverage files are updated and that pollutes your git commit history with files that are changed but aren't "real" changes in the project.

As long as simplecov is configured, anyone who downloads your project and runs the tests will generate a coverage report that they can view.